runtime/common/src/gov_origin.rs

```diff
@@ -0,0 +1,23 @@
+// --- paritytech ---
+use frame_system::{EnsureOneOf, EnsureRoot};
+use pallet_collective::{EnsureProportionAtLeast, EnsureProportionMoreThan};
+use sp_core::u32_trait::{_1, _2, _3, _5};
+// --- darwinia-network ---
+use darwinia_primitives::AccountId;
+
+pub type Root = EnsureRoot<AccountId>;
+
+pub type AtLeastHalf<Collective> =
```

Comment:
```suggestion
pub type RootOrAtLeastHalf<Collective> =
```

Reply:
Root can do almost everything. So I remove it from the name before.